### ISSUE TITLE
[FEATURE] Improve accesibility for the Modal Box

### ISSUE THREAD

**Issue by Chandraprakash-Darji** [EddieHubCommunity/LinkFree#5912]
### Description

I have shared the screenshot of the page, I request the feature accessibility of the Modal box should be improved. 

1. When we click ESC then it should close the modal but is not closing it
2. When we click outside of Modal box it should close it is not.

Visit: https://linkfree.eddiehub.io/playground

### Screenshots

![Screenshot 2023-04-08 at 11 21 44 AM](https://user-images.githubusercontent.com/93640141/230705746-a4207d03-f81c-45aa-a3a9-b5fc2f308106.png)

**Comment by eddiejaoude** [comment]
Good spot @Chandraprakash-Darji , I see we have an issue with dark mode and you are also suggesting some improvements

Please can you rename this issue to only focus on the dark mode issue for the model, and then create another issue for the modal improvements

**Comment by eddiejaoude** [comment]
I just saw issue https://github.com/EddieHubCommunity/LinkFree/issues/5911 please ignore my comment above - this issue is focused on the improvements to the model

**Comment by Chandraprakash-Darji** [comment]
@eddiejaoude I would love to work on this issue? If you can assign

**Comment by NiazMorshed2007** [comment]
Hey @Chandraprakash-Darji, just saw you are assigned to #5911  issue. Can I take on this?

**Comment by Chandraprakash-Darji** [comment]
@NiazMorshed2007 sure go ahead

**Comment by sanskar1306** [comment]
I would love to work on this issue.

**Comment by NiazMorshed2007** [comment]
@sanskar1306 sorry mate :(, I've already done it, I'm about to make a pr now and just see your comment

**Comment by MAYANKpandey14** [comment]
> I would love to work on this issue.

Hey, @sanskar1306  really appreciate your will for Contribution. However, it has been already been assigned to Niaz . There are many other issue for you to contribute. 
Welcome to the community , feel free to ask any question.

**Comment by NiazMorshed2007** [comment]
@eddiejaoude I'll keep in mind next time. Thanks 🙏

**Comment by eddiejaoude** [comment]
Merged https://github.com/EddieHubCommunity/LinkFree/pull/5932
